dipy/reconst/qtdmri.py

```diff
@@ -1105,20 +1113,48 @@ def qtdmri_to_mapmri_matrix(radial_order, time_order, ut, tau):
 
     counter = 0
     mapmri_mat = np.zeros((n_elem_mapmri, n_elem_qtdmri))
-    for nxt, nyt, nzt, o in qtdmri_ind_mat:
-        index_overlap = np.all([nxt == mapmri_ind_mat[:, 0],
-                                nyt == mapmri_ind_mat[:, 1],
-                                nzt == mapmri_ind_mat[:, 2]], 0)
+    for j, ll, m, o in qtdmri_ind_mat:
+        index_overlap = np.all([j == mapmri_ind_mat[:, 0],
+                                ll == mapmri_ind_mat[:, 1],
+                                m == mapmri_ind_mat[:, 2]], 0)
         mapmri_mat[:, counter] = temporal_storage[o] * index_overlap
         counter += 1
     return mapmri_mat
 
 
+def qtdmri_to_mapmri_matrix(radial_order, time_order, ut, tau):
+    """Generates the matrix that maps the qtdmri coefficients to MAP-MRI
```

Comment:
Can you update the docstring to be in imperative mood.  `Generate` instead of `Generates`